dev/devicelab/bin/run.dart

```diff
@@ -101,6 +101,13 @@ Future<void> main(List<String> rawArgs) async {
     return;
   }
 
+  final String/*?*/ logPath = args['device-logs-path'] as String/*?*/;
```

Comment:
The arg is `device-log-path`.  Probably worth adding `device-logs-path` as an alias to the arg if you, the person who wrote it a few minutes ago, couldn't quite remember which it was 😄

Reply:
Yes...

Reply:
WDYT of doing this as an optional arg vs. detecting `LocalPlatform().environment[FLUTTER_LOGS_DIR]`?

I mentionedin the issue, I'd rather avoid the `HostAgent` logic since I don't want to do this by default on local dev machines.

Reply:
As discussed in chat, uploaded a new change to make hostAGent not fall back to a local temp dir (and updated the test to make sure that's the case), and now using hostAgent.